src/yang/example-compa-diffserv.yang

```diff
@@ -3,54 +3,54 @@ module example-compa-diffserv {
   namespace "urn:ietf:params:xml:ns:yang:example-compa-diffserv";
   prefix example;
 
-  import ietf-qos-classifier {
-    prefix classifier;
-    reference "RFC XXXX: YANG Model for QoS";
+  import iana-qos-types {
+    prefix qos-types;
+    reference
+      "RFC XXXX: YANG Models for Quality of Service (QoS).";
   }
-  import ietf-qos-policy {
+  import ietf-traffic-policy {
     prefix policy;
-    reference "RFC XXXX: YANG Model for QoS";
+    reference
+      "RFC XXXX: YANG Models for Quality of Service (QoS).";
   }
   import ietf-qos-action {
     prefix action;
-    reference "RFC XXXX: YANG Model for QoS";
+    reference
+      "RFC XXXX: YANG Models for Quality of Service (QoS).";
   }
   import ietf-diffserv {
     prefix diffserv;
-    reference "RFC XXXX: YANG Model for QoS";
+    reference
+      "RFC XXXX: YANG Models for Quality of Service (QoS).";
   }
-
-  organization
-    "Company A";
+  organization "Company A";
   contact
     "Editor:   XYZ
-               <mailto:xyz@compa.com>";
+              <mailto:xyz@compa.com>";
   description
     "This module contains a collection of YANG definitions of
      companyA diffserv specification extension.";
-
   revision YYYY-MM-DD {
     description
       "Initial revision for diffserv actions on network packets";
     reference
-      "RFC 6020: YANG - A Data Modeling Language for the
-                 Network Configuration Protocol (NETCONF)";
+      "RFC 7950: The YANG 1.1 Data Modeling Language";
```

Comment:
Not sure why the YANG RFC is being referenced here. Should be 

RFC XXXX: A YANG Data Model for Quality of Service (QoS).

Reply:
HI Mahesh, how are you compiling it? I did 'make', I saw only one error not related to these changes.. I will fix this RFC reference.

Reply:
Ahh! Yes. I had enabled validation of examples for compa, compb, and compc, and that was displaying the errors. We can address them in a separate PR.

Once you have updated the reference, I can go ahead and approve these changes.

Reply:
Ok, I have taken care of example a and b files and the RFC reference.. We can handle example c separately.